frontend/src/store/episode/reducer.ts

```diff
@@ -37,6 +37,9 @@ const reducer = createReducer(initialState, (builder) => {
   builder.addCase(loadCommentsByEpisodeId.rejected, (state) => {
     state.dataStatus = DataStatus.REJECTED;
   });
+  builder.addCase(createComment.fulfilled, (state, action) => {
+    state.comments.push(action.payload);
+  });
```

Comment:
you need to add `commentDataStatus` to the reducer, and disable the form while creating comment